factionParser.py

```diff
@@ -0,0 +1,70 @@
+import os
+import sys
+import json
+from pprint import pp
+
+def process_Faction_Collection(file_path):
+    with open(file_path, "r") as json_file:
+        data = json.load(json_file)
+        item_name = data[0].get("items")
+        return item_name
+
+def process_files(primary_path, prefix):
+    file_dict = {}
+    csv_files_index = index_csv_files("DynamicShops/")
+    # Iterate through files in primary path and process each
+    for file_name in os.listdir(primary_path):
+        full_path = os.path.join(primary_path, file_name)
+        if os.path.isfile(full_path):
+            file_dict[file_name] = []
+            if file_name.endswith(".json"):
+                collection_name = process_Faction_Collection(full_path)
+
+                add_file_contents(collection_name, csv_files_index, file_dict, file_dict[file_name], prefix)
+
+    return file_dict
+
+def add_file_contents(collection_name, csv_files_index, file_dict, current_file_lines, prefix):
+        # Read file and get its contents
+        csv_files_index[collection_name]
+        #print(csv_files_index)
+        with open(csv_files_index[collection_name], 'r') as file:
```

Comment:
if the headers were meaningful and we needed other fields, we could use `Csv.DictReader` here. [Python Docs - CSV](https://docs.python.org/3/library/csv.html#csv.DictReader)